packages/url_launcher/url_launcher/README.md

```diff
@@ -8,6 +8,22 @@ iOS, Android, web, Windows, macOS, and Linux.
 ## Usage
 To use this plugin, add `url_launcher` as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
 
+## Installation
+
+### iOS 
+Add any URL scheme passed to canLaunch in your Info.plist file.
```

Comment:
Nits:
- Add `s around canLaunch
- s/in your/as `LSApplicationQueriesSchemes` entries in your/